src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorTest.cs

```diff
@@ -69,8 +69,8 @@ public void AddTimeData_WithSmallSampleSet_CalculatedCorrectly()
                 DateTime adjustedTime = dateTimeProvider.GetAdjustedTime();
                 DateTime normalTime = dateTimeProvider.GetUtcNow();
                 TimeSpan diff = adjustedTime - normalTime;
-
-                Assert.True(Math.Abs(diff.TotalMilliseconds) < expectedTimeOffsetLessThanMs + TimeEpsilonMs);
+                  
+                Assert.True(Math.Abs(diff.TotalMilliseconds) < expectedTimeOffsetLessThanMs + TimeEpsilonMs, $"Failed in sample at index:{i}. Actual offset milliseconds: {diff.TotalMilliseconds}. Expected offset milliseconds:{expectedTimeOffsetLessThanMs}");
```

Comment:
fix consistency of using spaces
I'd say `Failed in sample at index {i}.` without `:` as in normal sentence
also missing `.` at the end

Reply:
agree with the spaces inconsistency but the : would be more consistent to keep in after the index.  I normally find it a good idea to have the colon just to know the value coming after it is dynamic, especially in cases where it can be blank, although in this case it can't.